### PR TITLE
[ fix ] isue 2745

### PR DESCRIPTION
**Pull request by stefan-hoeck**
As discussed in #2746, this is a more straightforward fix for #2745. I added also a new test to make sure Idris can find the `.ipkg` files it installed.

### CODE REVIEW

File: tests/idris2/pkg016/run [hunk 1]
```diff
@@ -0,0 +1,13 @@
+IDRIS2_VERSION="idris2-0.6.0"
+
+$1 --install foo.ipkg > /dev/null
+$1 --install bar.ipkg > /dev/null
+$1 --install baz.ipkg > /dev/null
+$1 --build test.ipkg
+
+build/exec/test
+
+rm -r "${IDRIS2_PREFIX}/${IDRIS2_VERSION}/foo-0"
```
Comment (gallais) [inline tests/idris2/pkg016/run:10]: Can't you write `rm -r "${IDRIS2_PREFIX}/idris2-*/foo-0"`?

**Comment by Z-snails** [comment]
I don't know if this would simplify it at al, but you could probably specify set `IDRIS2_PREFIX` in the test (pkg016), and use `--no-prelude` so it doesn't complain about missing prelude. Then it definitely won't mess with any global install.

**Comment by stefan-hoeck** [comment]
> I don't know if this would simplify it at al, but you could probably specify set `IDRIS2_PREFIX` in the test (pkg016), and use `--no-prelude` so it doesn't complain about missing prelude. Then it definitely won't mess with any global install.

Yeah, in that case I'd have to change how the test is being run (it's using a `main` function at the moment). Personally, I think the current solution is reasonably simple.

**Comment by Z-snails** [comment]
Ok I'll merge it as is :)

**Pull request merged**